### ISSUE TITLE
App tagline and description for Urbit is not logical for those that don't already know what it is

### ISSUE THREAD

**Issue by johnpc** [getumbrel/umbrel-apps#1773]
See https://github.com/getumbrel/umbrel-apps/blob/master/urbit/umbrel-app.yml#L7-L20

```
tagline: Run Urbit on your Umbrel
description: >-
  Urbit is a personal server for self-sovereign personal & networked
  computing. Nock, a functional combinator, is built into Hoon, a system
  programming language, used to implement Arvo, a deterministic operating
  system, that communicates over Ames, a decentralized, encrypted P2P network.


  This app helps you boot and manage Urbit instances on your Umbrel. It provides a simple management GUI, start and stop operations, and passes your +code through the interface.


  If you are interested in purchasing an L2 planet checkout: subject.network/buy


  Credit to ~timluc-miptev, ~master-forwex, ~sipsen-pilser & ~rivpyl-sidfyl
  ```

Perhaps if you already know what Urbit is, this circular-definition of a tagline and jargon-fueled description make sense. But people who already know what it is don't need the description. The description should be grokkable by people who don't already know what Urbit is (like me!).

Since I don't know what it is and have never used it, I am not qualified to write the tagline or description. However here is an AI generated one that feels like it makes more sense:

```
tagline: Own your data, run your apps, and connect with others on your personal digital server
description: >-
 Urbit is a new way of using computers and the internet that tries to give people more control over their digital 
 lives. Instead of relying on 
 big companies to store your information and run your apps, Urbit lets you do all of this on your own "digital 
 space." This space is called a "ship," and it's like your own private island on the internet. You can use it to talk 
 to friends, store files, and run apps without worrying about your data being used by others. Urbit is built from 
 scratch to be more secure and private than the regular internet. While it's still being developed and improved, 
 the goal of Urbit is to create an internet where people have more freedom and control over their online 
 activities.
```

Someone should update the description, and perhaps use the AI one here if it's correct.

**Comment by yapishu** [comment]
> this circular-definition of a tagline and jargon-fueled description make sense

sorry you didn't like it :(  i don't think it is a 'circular definition' at all though, it is a concise technical description of a very unusual and widely scoped project, followed by a short description of what the app does. urbit is an entirely novel computing stack. though i agree it  could probably use more emphasis on the social aspects since that's how most people use it

**Comment by johnpc** [comment]
> Name: Urbit
> Description: This app helps you boot and manage Urbit instances on your Umbrel

That's circular by definition because it uses the word it's defining in its own definition. It would be better if the description mentioned the problems it solved instead of rattling off the technologies it uses. From this description it could be for shopping, p2p file sharing, p2p communication, network monitoring, or literally anything. There is nothing about the description that that indicates anything about it wants to do, other than "use Hoon" and "use Ames" and "implement Avro" which are cryptic and unhelpful.

**Comment by johnpc** [comment]
Imagine if the Uptime Kuma description was "Built with Vue, offers Gotify and HTTP(s) Json Query for arbitrary computing stacks" without ever mentioning that it's a notification service for application downtimes. I think we all agree that would be insane. This is the equivalent.

**Comment by yapishu** [comment]
> From this description it could be for shopping, p2p file sharing, p2p communication, network monitoring, or literally anything. 

yes, urbit is a general purpose computer which can be programmed to do things that computers can do.

**Comment by johnpc** [comment]
I just think having some type of "why" explained would be nice. But maybe it's just me that doesn't get it ¯\_(ツ)_/¯
